flink-runtime/src/main/java/org/apache/flink/runtime/taskexecutor/TaskExecutor.java

```diff
@@ -1413,7 +1413,8 @@ public ResourceID getResourceID() {
 	 *
 	 * @param t The exception describing the fatal error
 	 */
-	void onFatalError(final Throwable t) {
+	@VisibleForTesting
+	public void onFatalError(final Throwable t) {
```

Comment:
I don't find a use point of onFatalError outside the package in your diff. Could you revert it?

Reply:
@TisonKun The method is used in TaskExecutorTests.testTerminationOnFatalError().

Reply:
But they are in the same package, aren't they?

Reply:
Thanks, I didn't notice that. I'll revert it.